### ISSUE TITLE
IE8 not work!

### ISSUE THREAD

**Issue by wuyuweixin** [KingSora/KingSora.github.io#5]
The scrollbar not change with IE8.

**Comment by KingSora** [comment]
I'll fix this in the next version, thanks! Meanwhile you can use `v1.7.1`, it works in IE8.

**Comment by wuyuweixin** [comment]
IE8 not support rgba.
![G Y`8WF~@04{}9S)T{W UOG](https://user-images.githubusercontent.com/16173597/59558961-1b28cf00-9030-11e9-9c13-1048f611b312.png)

**Comment by KingSora** [comment]
The RGBA thing was patched in `v1.4.x` (or so), you have to adjust it by yourself... the textarea issue isn't happening on my end, can you please upload your example so I can test it? Thanks!

**Comment by wuyuweixin** [comment]
[https://github.com/wuyuweixin/custom-scrollbar](https://github.com/wuyuweixin/custom-scrollbar)

**Comment by wuyuweixin** [comment]
@KingSora thank you for your time spent on this issue.

**Comment by KingSora** [comment]
@wuyuweixin no problem, please try to load only one non-minified version: either the `jQuery` version **OR** the `vanilla` version but not both. I think this is causing some trouble in IE8. 

Currently you load them like:
```html
<script src="js/jquery.overlayScrollbars.min.js"></script>
<script src="js/OverlayScrollbars.min.js"></script>
```
Change it to:
```html
<!--<script src="js/jquery.overlayScrollbars.min.js"></script>-->
<script src="js/OverlayScrollbars.js"></script>
```

**Comment by wuyuweixin** [comment]
Thank you very much !

**Comment by KingSora** [comment]
@wuyuweixin The `IE8` bug was fixed in [v1.7.3](https://github.com/KingSora/OverlayScrollbars/releases/tag/v1.7.3).
